index.html

```diff
@@ -0,0 +1,330 @@
+        </div>
+
+        <div class="my-work">
+          <img src="image/works/SnapshootPortfolio4.svg" />
+          <div class="my-work-body">
+            <h2>Multi-Post Stories</h2>
+            <div class="position-container">
+              <span>Canopy</span>
+              <ul>
+                <li class="developed marked">Backend development</li>
+                <li class="year marked">2018</li>
+              </ul>
+            </div>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="tech">
+              <li>html</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+            <a href="#">See Project</a>
+          </div>
+        </div>
+      </section>
+      <!-- section 2 ends Here -->
+
+      <!-- section 3 starts here -->
+      <section class="section3">
+        <div class="about">
+          <h1>About Myself</h1>
+          <p>
+            Hello I’m a software developer! I can help you build a product,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+
+          <div class="social"><p>LET’S CONNECT</p></div>
+
+          <div class="icons">
+            <ul>
+              <li>
+                <a href="#"><i class="fa fa-facebook fa-2x"></i></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa fa-twitter fa-2x"></i></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa fa-instagram fa-2x"></i></a>
+              </li>
+
+              <li>
+                <a href="#"><i class="fa fa-github fa-2x"></i></a>
+              </li>
+
+              <li>
+                <a href="#"><i class="fa fa-linkedin fa-2x"></i></a>
+              </li>
+            </ul>
+          </div>
+          <div class="get-resume">
+            <button type="button" class="get-resume-btn">Get My Resume</button>
+          </div>
+        </div>
+
+        <div class="accordion">
+          <ul class="mother-accordion">
+            <li class="accordion-item">
+              <h2>
+                <span>Languages</span>
+                <i class="fa fa-angle-down" aria-hidden="true"></i>
+              </h2>
+              <ul class="child-accordion">
+                <li>
+                  <div class="language-name">
+                    <img src="./image/language/js.svg" alt="" />
+                    <span>JavaSript</span>
+                  </div>
+                </li>
+                <li>
+                  <div class="language-name">
+                    <img src="./image/language/html.svg" alt="" />
+                    <span>HTML</span>
+                  </div>
+                </li>
+                <li>
+                  <div class="language-name">
```

Comment:
- [ ] kindly remove the border-bottom for the first child of the `<ul>` to match figma design


![image](https://user-images.githubusercontent.com/89282221/193110503-d9a98459-b0f0-42fd-bd02-c6cca0691840.png)